README.markdown

```diff
@@ -10,220 +10,134 @@ There are two ways in which you can install OneZoom on a personal computer: full
 
 * *Full installation* creates an entire duplicate of the OneZoom website, which is built using the [web2py](http://web2py.com) framework. This creates a fully self-contained local system (apart from the picture files, which can be downloaded separately). This is the most reliable installation method, but requires you to install and run extra software packages, in particular [web2py](http://web2py.com) and a [MySQL](https://www.mysql.com) server. Since this can be quite complicated, the majority of this readme contains instructions for full installation.
 
-
```

Comment:
Can't comment on existing lines, but in first paragraph it links to `README_SERVER.markdown`, which doesn't seem to exist.